doc/release-notes/7440-multilicense.md

```diff
@@ -0,0 +1,119 @@
+### Multiple License Support
+
+Users can now select from a set of configured licenses in addition to or instead of the current Creative Commons CC0 choice or provide custom terms of use (if configured) for their datasets. Administrators can configure their Dataverse instance via API to allow any desired license as a choice and can enable/disable the option to allow custom terms. Administrators can also mark licenses as 'inactive' to disallow future use while keeping that license for existing datasets. By default, only the CC0 license will be preinstalled. Examples in the Guides show how to add additional licenses and specific examples are given for several Creative Commons licenses. **Note: Datasets in existing installations will automatically be updated to conform to new requirements that custom terms cannot be used with a standard license and that custom terms cannot be empty. Administrators may wish to manually update datasets with these conditions if they do not like the automated migration choices. See the Notes for Dataverse Installation Administrators and Additional Release Steps sections for further information.**
```

Comment:
We didn't want to add for existing installations since that's a policy change, but having more (e.g. the basic CC choices) for new installations could make sense, although I'm not sure - even then admins may want to review before allowing anything beyond fully public/CC0. 

(I'm also not sure how best to do it - would it be a change in the installer to call the API after Dataverse is running? Or is there a better way? (Can the mechanism that autocreates tables for new installs also add rows (and not affect existing tables?). Ansible? (It might be nice to preconfigure with the standard GDCC previewers too - another set of API calls that could/should only run on new installs))

In any case, if there's consensus on changes we can do that here or as part of a new issue/PR.

Reply:
Yes, exactly. If we want the product to ship with half a dozen CC licenses, we'd create the JSON files and have the installer create each license. This would be for new installations.

For existing installations, we'd just say in the release note that new licenses are possible and probably point out the JSON files that new installations wlll have loaded if they want to have the same licenses as out of the box Dataverse. I don't think we should populate existing installations with licenses. Let them opt in.

Obviously, yes, this could all be done after this pull request is merged in a new pull request, but I thought I'd raise it now. Otherwise, it feels like a bit of a hidden feature that installations can have a nice dropdown of popular licenses.